GameData/RealismOverhaul/Engine_Configs/Agena_XLR81_Config.cfg

```diff
@@ -377,8 +384,8 @@
 			ullage			= False //(1) p 1-4
```

Comment:
Let's avoid `tab`-alignment IMHO. Clutters the diff and can be wrong depending on tab size.